.github/pages/hello-world.md

```diff
@@ -35,6 +35,51 @@ This downloads the Fashion-MNIST training and test dataset and tells Jina to ind
 Then it randomly samples images from the test set as queries and asks Jina to retrieve relevant results.
 The whole process takes about 1 minute.
 
+<br><br>
+
+#### Use Jina Hub Executors
+
+
+You can run the `jina hello fashion` demo using a different embedding method.
+To do so:
+
+- 1) Clone the repository with  `jina hello fork fashion <your_project_folder>`.
+In `your_project_folder` you will  have a file `app.py`  that you can change to leverage other embedding methods. 
+
+- 2) Change  lines 74 to 79 from `app.py` to define a different `Flow`. For example, you can use  [ImageTorchEncoder](https://github.com/jina-ai/executor-image-torch-encoder)
+  changing 
+  
+      ```python
+     f = (
+          Flow()
+          .add(uses=MyEncoder, parallel=2)
+          .add(uses=MyIndexer, workspace=args.workdir)
+          .add(uses=MyEvaluator)
+          )
+      ```
+
+     with the  flow
+  
+     ```python
+     f = (
+         Flow()
+         .add(uses='jinahub+docker://ImageTorchEncoder',
+              override_with={'model_name': 'alexnet'},
+              parallel=1)
```

Comment:
can u check if this works with parallel 2?

Reply:
It seems to  work

Reply:
then we can keep `args.parallel` then no?